src/pages/workspaces/WorkspacesList/WorkspacesListTabs.test.ts

```diff
@@ -0,0 +1,118 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+import { updateSearch, useRoute } from 'src/libs/nav';
+import { CatagorizedWorkspaces } from 'src/pages/workspaces/WorkspacesList/CatagorizedWorkspaces';
+import { WorkspacesListTabs } from 'src/pages/workspaces/WorkspacesList/WorkspacesListTabs';
+import { asMockedFn } from 'src/testing/test-utils';
+import { defaultAzureWorkspace, defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+
+// the FlexTable uses react-virtualized's AutoSizer to size the table.
+// This makes the virtualized window large enough for all rows/columns to be rendered in tests.
+jest.mock('react-virtualized', () => ({
```

Comment:
please add type safety to these mocked modules....
```

type TheModuleExports = typeof import('path/themodule');
jest.mock('path/themodule', (): TheModuleExports => {
  //...
}
```

and add a cast to jest.requireActual calls:

`   ...jest.requireActual<TheModuleExports>('path/themodule')`